x-pack/platform/plugins/shared/alerting/common/routes/schedule/schema/v1.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import {
+  validateStartDateV1,
+  validateEndDateV1,
+  validateIntervalAndFrequencyV1,
+  validateOnWeekDayV1,
+  validateDurationV1,
+  validateTimezoneV1,
+  validateScheduleV1,
+} from '../validation';
+
+export const scheduleRequestSchema = schema.object(
```

Comment:
This schema is intended to be used in other places, so `snooze` shouldn't be part of the documentation descriptions. 😄